Installation/cmake/modules/FindOpenMesh.cmake

```
@@ -44,7 +46,34 @@ endif()
 
 include( FindPackageHandleStandardArgs )
 
-FIND_PACKAGE_HANDLE_STANDARD_ARGS(OpenMesh
-        REQUIRED_VARS OPENMESH_INCLUDE_DIR OPENMESH_LIBRARIES
-        FOUND_VAR OpenMesh_FOUND
-        )
+find_package_handle_standard_args(OpenMesh
+  REQUIRED_VARS OPENMESH_INCLUDE_DIR OPENMESH_LIBRARIES
+  FOUND_VAR OpenMesh_FOUND
+  )
+
+if(OpenMesh_FOUND AND NOT TARGET OpenMesh::OpenMesh)
+  add_library(OpenMesh::OpenMesh UNKNOWN IMPORTED)
+
+  if(TARGET OpenMeshCore)
+    target_link_libraries(OpenMesh::OpenMesh PUBLIC OpenMeshCore)
+    return()
+  endif()
+
+  set_target_properties(OpenMesh::OpenMesh PROPERTIES
+    INTERFACE_COMPILE_DEFINITIONS "CGAL_USE_OPENMESH;NOMINMAX;_USE_MATH_DEFINES"
+    INTERFACE_INCLUDE_DIRECTORIES  "${OPENMESH_INCLUDE_DIRS}")
```

There was a type in my previous commits:
```suggestion
    INTERFACE_INCLUDE_DIRECTORIES  "${OPENMESH_INCLUDE_DIR}")
```
I will fix it as soon as I know which version of the branch is the right one.